examples/basic/main.cpp

```diff
@@ -13,7 +13,7 @@
 #define STB_IMAGE_WRITE_IMPLEMENTATION
 #define TINYGLTF_NOEXCEPTION
 #define JSON_NOEXCEPTION
-#include "../../tiny_gltf.h"
+#include "../../include/tiny_gltf.h"
```

Comment:
not super important but I'm surprised that `tiny_gltf.h` wasn't found in a search path before exactly specifying the relative path`

Reply:
clion made the change for me when I drag & dropped in the GUI

Reply:
oh interesting.

Reply:
should this change be dropped then?

Reply:
Yeah, dropped it in both places that it happened